src/App.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
+import BooksList from './components/BooksList';
+import BookForm from './components/BookForm';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Hello</h1>
-    </div>
-  );
-}
+const App = () => (
```

Comment:
Move this component to src/component
At this moment App is not having any logic, let us move it to it right place.